<?xml version="1.0" encoding="UTF-8"?>
<webMethodsTestSuite description="" name="TS_wmTestSuite.xml">
    <webMethodsTestCase name="StraightTest">
        <service folder="test1" name="divideInts">
            <input>
                <file filename="resources/data/input_divideInt_StraightTest.xml"/>
            </input>
            <expected>
                <file filename="resources/data/output_divideInt_StraightTest.xml"/>
            </expected>
        </service>
    </webMethodsTestCase>
    <webMethodsTestCase name="ExceptionTest">
        <service folder="test1" name="divideInts">
            <input>
                <file filename="resources/data/input_divideInt_Exception.xml"/>
            </input>
            <expected>
                <exception class="java.lang.ArithmeticException"/>
            </expected>
        </service>
    </webMethodsTestCase>
</webMethodsTestSuite>
